GX Password/strings/th/Resources.resw

```diff
@@ -0,0 +1,162 @@
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="DescApp" xml:space="preserve">
+    <value>เครื่องมือสร้างรหัสผ่านที่ปลอดภัย ง่าย และรวดเร็ว!</value>
+  </data>
+  <data name="passLeng.Text" xml:space="preserve">
+    <value>ความยาวของรหัสผ่าน:</value>
+  </data>
+  <data name="passOpt.Text" xml:space="preserve">
+    <value>ตัวเลือกรหัสผ่าน:</value>
+  </data>
+  <data name="sBtClipboard.Content" xml:space="preserve">
+    <value>คลิปบอร์ด</value>
+  </data>
+  <data name="sBtOk.Content" xml:space="preserve">
+    <value>สร้างรหัสผ่านเลย</value>
+  </data>
+  <data name="sTsLett.OffContent" xml:space="preserve">
+    <value>มีตัวอักษร</value>
+  </data>
+  <data name="sTsLett.OnContent" xml:space="preserve">
+    <value>ไม่มีตัวอักษร</value>
+  </data>
+  <data name="sTsNumb.OffContent" xml:space="preserve">
+    <value>ไม่มีตัวเลข</value>
+  </data>
+  <data name="sTsNumb.OnContent" xml:space="preserve">
+    <value>มีตัวเลข</value>
+  </data>
+  <data name="sTsSimi.OffContent" xml:space="preserve">
+    <value>ไม่มีอักขระซ้ำ</value>
+  </data>
+  <data name="sTsSimi.OnContent" xml:space="preserve">
+    <value>มีอักขระซ้ำกันได้</value>
+  </data>
+  <data name="sTsSymb.OffContent" xml:space="preserve">
+    <value>ไม่มีสัญลักษณ์พิเศษ</value>
+  </data>
+  <data name="sTsSymb.OnContent" xml:space="preserve">
+    <value>มีสัญลักษณ์พิเศษ</value>
+  </data>
+  <data name="yourPass.Text" xml:space="preserve">
+    <value>รหัสผ่านใหม่ของคุณ! :</value>

```

Comment:
Empty space ?

Reply:
Thais like excitement on using an app so I put "!" at the end and the space makes it looks prettier after "!" thing, hahah. but you can remove it if you'd like to ;)